pandas/core/resample.py

```diff
@@ -624,18 +624,134 @@ def backfill(self, limit=None):
 
     def fillna(self, method, limit=None):
         """
-        Fill missing values
+        Fill the new missing values in the resampled data using different
+        methods.
+
+        In statistics, imputation is the process of replacing missing data with
+        substituted values [1]_. When resampling data, missing values may
+        appear (e.g., when the resampling frequency is higher than the original
+        frequency).
+
+        The backward fill ('bfill') will replace NaN values that appeared in
+        the resampled data with the next value in the original sequence. The
+        forward fill ('ffill'), on the other hand, will replace NaN values
+        that appeared in the resampled data with the previous value in the
+        original sequence. Missing values that existed in the orginal data will
+        not be modified.
 
         Parameters
         ----------
         method : str, method of resampling ('ffill', 'bfill')
+            Method to use for filling holes in resampled data
+                * ffill: use previous valid observation to fill gap (forward
```

Comment:
No indentation is needed here (compared to the "Method ..." on the line above), but, sphinx needs a blank line between those two lines

Reply:
Done